app/(general)/integration/livepeer/livestream/page.tsx

```diff
@@ -0,0 +1,27 @@
+'use client'
+
+import { LivepeerConfig, ThemeConfig, createReactClient, studioProvider } from '@livepeer/react'
```

Comment:
`@livepeer/react` was not added to package.json, please install it into the repo